src/amp.js

```diff
@@ -0,0 +1,4 @@
+import * as aphrodite from 'aphrodite/no-important';
```

Comment:
That could be enabled on either as an option, or we could add a third entry point. I see value in having a no-important version of this in general.